sig-release/annual-report-2022.md

```diff
@@ -2,76 +2,105 @@
 
 4. If your group has special training, requirements for reviewers/approvers, or processes beyond the general [contributor guide],
    does your [CONTRIBUTING.md] document those to help **existing** contributors grow throughout the [contributor ladder]?
 
-   -
+   We have a lot of information, including detailed role handbooks for both subprojects:
+
+   - [Release Team](https://github.com/kubernetes/sig-release/tree/master/release-team)
+   - [Release Engineering](https://github.com/kubernetes/sig-release/tree/master/release-engineering)
+
+   However, these are not linked directly from CONTRIBUTING.md. We will use the [issue above](https://github.com/kubernetes/sig-release/issues/2200) to improve the discoverability of these. 
+
 
 5. Does the group have contributors from multiple companies/affiliations?
 
-   -
+   - Yes. Based on the last year of data from [dev stats](https://k8s.devstats.cncf.io/d/55/company-prs-in-repository-groups?orgId=1&var-period_name=Last%20year&var-repogroups=SIG%20Release&var-repos=All&var-companies=All&var-countries=All) we have had contributions from the following companies over the last year:
+      - Red Hat Inc,
+      - Chainguard Inc,
+      - Intel Corporation
+      - Liquid Reply
+      - Kubermatic GmbH
+      - Google LLC
+      - Microsoft Corporation
+      - Cisco
+      - Amazon
+      - VMware Inc
+      - SUSE LLC
+      - International Business Machines
+      - Jetstack LTD
+      - Mesosphere
+      - Mastercard International Incorporated
+      - DaoCloud Network Technology Co. Ltd.
+      - Oracle America Inc.
+      - Rackspace
+      - NEC Corporation
+
+   This data reflects company information in terms of PRs. We also have had contributions from individuals that have no company affiliation and several individuals from the CNCF.
+   In addition to code contributions, the release teams during 2022 were staffed by individuals from a wide range of corporations, as well as students and other independent individuals. These contributions are not all reflected by the devstats query above, but are important to recognize.
 
 6. Are there ways end users/companies can contribute that they currently are not?
    If one of those ways is more full time support, what would they work on and why?
 
-   -
-   -
+   - The release tooling is currently supported by the core of the Release Engineering team. There are opportunities for end users and Kubernetes distributors to support the maintenance of the tooling within our repositories, including the `bom` tool. Additionally, vendor companies that build and distribute their own Kubernetes releases could provide more support to SIG Release, specifically Release Engineering, in order to grow contributors that could help with important tasks like Go version updates. 
```

Comment:
Can we point to some docs/issues for these asks from end users/companies/vendors?

Reply:
I cleaned this up, we don't have specific docs/asks and I think this is specifically indicating full-time work, which I don't think we have direct need of currently.